requirements-dev.txt

```diff
@@ -18,6 +18,8 @@ pycodestyle==2.6.0
 pyflakes==2.2.0
 pyparsing==2.4.7
 pytest==6.1.2
+pytest-cov
```

Comment:
don't you want to pin the versions for reproducibility?

Reply:
Ah yes I can do that. Will update